pkg/queue/concurrency_state.go

```diff
@@ -85,3 +88,55 @@ func ConcurrencyStateHandler(logger *zap.SugaredLogger, h http.Handler, pause, r
 		<-done
 	}
 }
+
+// Pause sends to an endpoint when request counts drop to zero
+func Pause(endpoint string) error {
+	action := ConcurrencyStateMessageBody{Action: "pause"}
+	bodyText, err := json.Marshal(action)
+	if err != nil {
+		return fmt.Errorf("unable to create request body: %w", err)
+	}
```

Comment:
Since this is static content anyway, can we cache it? Seems wasteful to marshal this on each request (same below)